packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt

```diff
@@ -85,8 +90,10 @@ internal class DataGripMongoDbDriver(
         ) {
             runQuery(
                 """
-                db.getSiblingDB("${database.encodeForJs()}")
-                  .runCommand(EJSON.parse("${command.toJson()}"))
+                EJSON.serialize(
```

Comment:
Is there a follow-up ticket to use regular BSON?

Reply:
Yep, we have INTELLIJ-35.